### ISSUE TITLE
Rename Site and project to MWM.DB

### ISSUE THREAD

**Issue by eyefox47** [eyefox47/dally-stats#2]
Should be changed:
- Repo name
- folder names
- ~Django app names~
- Header
- Titlte bar

**Comment by eyefox47** [comment]
Still not sure how the exact name of the database site should be. AFAIK the tumblr will use 

> Mistakes Were Made...

as a title. Ideally it should also work with a subtitle. Options could be:

- Mistakes Were Made DB [img](https://cloud.githubusercontent.com/assets/6519072/23032938/830d7d94-f476-11e6-9bf1-681f6ebb4177.png)
- Mistakes Were Made.DB [img](https://cloud.githubusercontent.com/assets/6519072/23032910/72bf3568-f476-11e6-851e-71f5fa470d08.png)
- **Mistakes Were Made...DB** [img](https://cloud.githubusercontent.com/assets/6519072/23032893/6397a98a-f476-11e6-8401-5a945fcade40.png)

Short version (will also be the name of the repository and Django project):
- MWM-DB
- **mwm-db**
- mwm_db

(edit) taking highlighted ones. The title on the website itself can easily be changed
